File: app/src/main/java/com/oxodiceproductions/dockmaker/Database/AppDatabase.java
package com.oxodiceproductions.dockmaker.Database;

import android.content.Context;

import androidx.room.Database;
import androidx.room.Room;
import androidx.room.RoomDatabase;

@Database(entities = {Document.class, Image.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract DocumentDao documentDao();

    public abstract ImageDao imageDao();

    public static AppDatabase getInstance(Context context){
        return Room.databaseBuilder(context,AppDatabase.class,"DocMakerRoomDB").build();
    }
}

File: app/src/main/java/com/oxodiceproductions/dockmaker/database/AppDatabase.kt
package com.oxodiceproductions.dockmaker.database

import android.content.Context
import androidx.room.Database
import androidx.room.Room
import androidx.room.RoomDatabase

@Database(entities = [Document::class, Image::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun documentDao(): DocumentDao
    abstract fun imageDao(): ImageDao

    companion object {
        fun getInstance(context: Context?): AppDatabase {
            return Room.databaseBuilder(
                context!!,
                AppDatabase::class.java, "DocMakerRoomDB"
            ).build()
        }
    }
}